holidays/countries/dr_congo.py

```diff
@@ -0,0 +1,92 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class DRCongo(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """Democratic Republic of the Congo holidays.
+
+    References:
+        * Public holidays in the Democratic Republic of the Congo <https://en.wikipedia.org/wiki/Public_holidays_in_the_Democratic_Republic_of_the_Congo>
+        * ORDONNANCE 79-154 <https://web.archive.org/web/20220329181351/http://www.leganet.cd/Legislation/DroitSocial/O.79.154.23.06.1979.htm>
+        * Ordonnance n° 14/010 <https://web.archive.org/web/20230419184344/http://leganet.cd/Legislation/Divers/Ordonnance.14.10.14.mai.2014.htm>
+        * Ordonnance n° 23-042 <https://web.archive.org/web/20250113230411/http://www.droitcongolais.info/files/143.03.23_Ordonnance-du-30-mars-2023_jours-feries.pdf>
+    """
+
+    country = "CD"
+    default_language = "fr"
+    supported_languages = ("en_US", "fr")
+    # Democratic Republic of the Congo marks independence from Belgium in June 1960.
+    start_year = 1961
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("Jour de l'An"))
+
+        # Martyrs of Independence.
+        self._add_holiday_jan_4(tr("Martyrs de l’indépendance"))
+
+        if self._year >= 2002:
+            # National Hero Laurent Désiré KABILA Day.
+            self._add_holiday_jan_16(tr("Journée du Héro National Laurent Désiré KABILA"))
+
+        if self._year >= 1962:
+            # National Hero Patrice Emery LUMUMBA Day.
+            self._add_holiday_jan_17(tr("Journée du Héro National Patrice Emery LUMUMBA"))
```

Comment:
We don't usually do full capitalization unless it's an acronym - also, I did look a bit into this and turns out there's no observance Pre-2015, and while [Loi 009-2002](https://www.leganet.cd/Legislation/Droit%20administratif/Urbanismevoiries/Div/L.009.05.08.2002.htm) establishes Order of National Heroes - there's no provision yet to declare these 2 dates holidays.

The same goes for Martyrs of Independencec Day as well
```suggestion
        # Established on May 10th, 2014 via Ordonnance n° 14/010.
        if self._year >= 2015:
            # Martyrs of Independence.
            self._add_holiday_jan_4(tr("Martyrs de l’indépendance"))

            # National Hero Laurent Désiré Kabila Day.
            self._add_holiday_jan_16(tr("Journée du héros national Laurent Désiré Kabila"))

            # National Hero Patrice Emery Lumumba Day.
            self._add_holiday_jan_17(tr("Journée du héros national Patrice Emery Lumumba"))
```